src/gui/kernel/qaction.cpp

```diff
@@ -14,6 +14,7 @@
 #endif
 #include <private/qguiapplication_p.h>
 #include <private/qdebug_p.h>
+#include <text/qtextdocument.h>
```

Comment:
included this to get access to `mightBeRichText`. there may be a better way to do this?